couchpotato/core/media/_base/providers/torrent/uhdbits.py

```diff
@@ -0,0 +1,259 @@
+import htmlentitydefs
+import json
+import re
+import time
+import traceback
+
+from couchpotato.core.helpers.encoding import tryUrlencode
+from couchpotato.core.helpers.variable import getTitle, tryInt, mergeDicts, getIdentifier
+from couchpotato.core.logger import CPLog
+from couchpotato.core.media._base.providers.torrent.base import TorrentProvider
+from dateutil.parser import parse
+import six
+
+
+log = CPLog(__name__)
+
+
+class Base(TorrentProvider):
+
+    urls = {
+        'login': 'https://uhdbits.org/login.php',
+        'login_check': 'https://uhdbits.org/inbox.php',
+        'api': 'https://uhdbits.org/ajax.php',
+        'detail': 'https://uhdbits.org/torrents.php'
+    }
+
+    login_errors = 0
+    http_time_between_calls = 2
+
+    def _search(self, media, quality, results):
```

Comment:
Please use the searchOnTitle function here, like https://github.com/CouchPotato/CouchPotatoServer/blob/master/couchpotato/core/media/_base/providers/nzb/omgwtfnzbs.py#L28
That function get's called multiple times with different title possibilities and does the work for you.